### PR TITLE
[W5.8b][W13-A2] Aadyaa Maddi

### PR DESCRIPTION
**Pull request by amad-person**
Added a couple of Non-Functional Requirements to the Developer Guide.

### CODE REVIEW

File: doc/DeveloperGuide.md [hunk 1]
```diff
@@ -94,6 +94,10 @@ Use case ends.
 2. Should be able to hold up to 1000 persons.
 3. Should come with automated unit tests and open source code.
 4. Should favor DOS style commands over Unix-style commands.
+5. Should be easy for a novice to use the application.
```
Comment (okkhoy) [inline doc/DeveloperGuide.md:97]: Define "easy"

File: doc/DeveloperGuide.md [hunk 2]
```diff
@@ -94,6 +94,10 @@ Use case ends.
...
+6. Should be convenient for users familiar with using a command-line interface.
```
Comment (okkhoy) [inline doc/DeveloperGuide.md:98]: Define "convenient" :stuck_out_tongue:

**Comment by okkhoy** [comment]
@amad-person 
NFR (and requirements in general) should be as precise as possible.
Avoid adjectives like `easy`, `fast`, `intuitive` etc.,
Instead, use precise statements like `response time should be 1 second`, `detailed help which shows all options should be present`

Some comments added. Please ack & close the PR after reading comments.

**Comment by amad-person** [comment]
Noted! Thanks

**Pull request closed without merging**